include/clap/ext/thread-check.h

```diff
@@ -8,9 +8,24 @@ static CLAP_CONSTEXPR const char CLAP_EXT_THREAD_CHECK[] = "clap.thread-check";
 extern "C" {
 #endif
 
+/* A note on threads as understood by CLAP:
+ *
+ * In the [main-thread], a CLAP plugin may carry out allocations, acquire a mutex and do IO,
```

Comment:
Add something like long running tasks such as indexing presets, ... should be run in background threads to keep the main thread responsive.